modules/node/content/cms_node.e

```diff
@@ -211,6 +217,41 @@ feature -- Element change
 			auther_set: author = u
 		end
 
+	mark_not_published
+			-- Set status to not_published
+		do
+			set_status ({CMS_NODE_CONSTANTS}.not_published)
```

Comment:
we could use a {CMS_NODE_API}.status_not_published: INTEGER = 1
without adding a new class.